packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -631,7 +480,15 @@ const ToggleMenuDrawer = observer(({ platform_config }) => {
                                 <ServerTime is_mobile />
                                 <NetworkStatus is_mobile />
                             </MobileDrawer.Footer>
-                            {is_language_change && <GetLanguageRoutesTraderHub />}
+                            {is_mobile_language_menu_open && (
+                                <MobileLanguageMenu
+                                    is_language_changing={is_language_changing}
+                                    expandSubMenu={expandSubMenu}
+                                    is_mobile_language_menu_open={is_mobile_language_menu_open}
+                                    setMobileLanguageMenuOpen={setMobileLanguageMenuOpen}
+                                    toggleDrawer={toggleDrawer}
```

Comment:
@yauheni-deriv Except `expandSubMenu` and `toggleDrawer` the rest are coming from `useStore`, You can just get them from the store instead of passing props.